sandboxed_containers/disabling-sandboxed-container-workloads.adoc

```diff
@@ -0,0 +1,8 @@
+[id="disabling-sandboxed-containers-workloads"]
+= Disabling OpenShift sandboxed containers workloads
```

Comment:
You can use your `OpenShift sandboxed containers` attribute here

Reply:
PV2/FCC Compliance - not to use attributes in the title